Add transparency information to social preview docs (#16418)
github/docs: content/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/customizing-your-repositorys-social-media-preview.md

    - To remove an image, click **Remove image**

    ![Social preview dropdown](/assets/images/help/repository/social-preview.png)

> ## Transparency

Reviewer: As per our content model
```suggestion
## About transparency
```

> PNGs with transparency are supported. Many communication platforms support a dark mode, so using a transparent social preview may be beneficial. The transparent image below is acceptable on a dark background; however, this is not always the case. When using an image with transparency keep in mind how it may look on different color backgrounds or platforms that don't support transparency.

Reviewer: Slightly rewording + avoiding passive voice + putting the advice given in the last sentence in its own paragraph to highlight it more.

```suggestion
We support PNG images with transparency. Many communication platforms support a dark mode, so using a transparent social preview may be beneficial. The transparent image below is acceptable on a dark background; however, this may not always be the case. 

When using an image with transparency, keep in mind how it may look on different color backgrounds or platforms that don't support transparency.
```

{% tip %}

> Tip: If you are not sure, we recommend using an image with a solid background.

Reviewer: Making this tip consistent with our other tips
```suggestion
**Tip:** If you aren't sure, we recommend using an image with a solid background.
```

    - To remove an image, click **Remove image**

    ![Social preview dropdown](/assets/images/help/repository/social-preview.png)

## About transparency

We support PNG images with transparency. Many communication platforms support a dark mode, so using a transparent social preview may be beneficial. The transparent image below is acceptable on a dark background; however, this may not always be the case. 

When using an image with transparency, keep in mind how it may look on different color backgrounds or platforms that don't support transparency.

> {% tip %}

Reviewer: ```suggestion
{% tip %}

```
